modules/docker/Makefile.build

```diff
@@ -0,0 +1,43 @@
+## Use DOCKER_IMAGE_NAME envvar to specify docker image with tags
+## User ARGS to pass arguments
+DOCKER_BUILD_PATH ?= .
+DOCKER_BUILD_FLAGS ?= --no-cache
+DOCKER_IMAGE_NAME ?= tests
+DOCKER_FILE ?= ./Dockerfile
+
+## Build docker image
+docker/build:
+	$(call assert-set,DOCKER)
+	$(call assert-set,DOCKER_IMAGE_NAME)
+	@BUILD_ARGS=`for arg in $$ARGS; do \
+		printf -- '--build-arg %s=%s ' "$$arg" "$${!arg}"; \
+	done`; \
+	echo "Building $(DOCKER_IMAGE_NAME) from $(DOCKER_FILE) with [$(DOCKER_BUILD_FLAGS)] build args..."; \
+	"$(DOCKER)" build $(DOCKER_BUILD_FLAGS) $$BUILD_ARGS -t $(DOCKER_IMAGE_NAME) -f $(DOCKER_FILE) $(DOCKER_BUILD_PATH)
+
+## Promote $SOURCE_DOCKER_REGISTRY/$IMAGE_NAME:$SOURCE_VERSION to $TARGET_DOCKER_REGISTRY/$IMAGE_NAME:$TARGET_VERSION
+docker/image/promote/local:
+	$(call assert-set,DOCKER)
+	$(call assert-set,IMAGE_NAME)
```

Comment:
What's the difference between `IMAGE_NAME` and `DOCKER_IMAGE_NAME`?

Reply:
Good catch, this was a typo. I'll take out `IMAGE_NAME`